helper-update-version.sh

```diff
@@ -1,16 +1,16 @@
 #!/bin/bash
 # Retrieve sources, get sha256sum, parse package version and update these strings in manifest, README and /conf/app.src
- 
-URL=$(curl -s https://api.github.com/repos/duniter/cesium/releases/latest | grep "browser_" | grep "web" | head -1 | cut -d\" -f4)
-wget -nc --quiet $URL -P ./tmp
-CHECKSUM=$(sha256sum ./tmp/cesium-*-web.zip | head -c 64)
-sed -i "s/SOURCE_SUM=.*/SOURCE_SUM=${CHECKSUM}/" ./conf/app.src
-sed -i "s#SOURCE_URL=.*#SOURCE_URL=$URL#" ./conf/app.src
 
-VERSION=$(echo $URL | sed "s#https://github.com/duniter/cesium/releases/download/\(.*\)/cesium.*#\1#")
-sed -i "s#\*\*Shipped version:\*\*.*#\*\*Shipped version:\*\* ${VERSION}#" ./README.md
-sed -i "s#    \"version\": \".*#    \"version\": \"${VERSION}\~ynh1\",#" ./manifest.json
+# URL=$(curl -s https://api.github.com/repos/duniter/cesium/releases/ | grep "browser_" | grep "web" | head -1 | cut -d\" -f4)
+# wget -nc --quiet $URL -P ./tmp
+# CHECKSUM=$(sha256sum ./tmp/cesium-*-web.zip | head -c 64)
+# sed -i "s/SOURCE_SUM=.*/SOURCE_SUM=${CHECKSUM}/" ./conf/app.src
+# sed -i "s#SOURCE_URL=.*#SOURCE_URL=$URL#" ./conf/app.src
 
-rm -f -R ./tmp
+# VERSION=$(echo $URL | sed "s#https://github.com/duniter/cesium/releases/download/\(.*\)/cesium.*#\1#")
+# sed -i "s#\*\*Shipped version:\*\*.*#\*\*Shipped version:\*\* ${VERSION}#" ./README.md
+# sed -i "s#    \"version\": \".*#    \"version\": \"${VERSION}\~ynh1\",#" ./manifest.json
 
-git commit README.md manifest.json conf/app.src -m "$VERSION"
```

Comment:
You should use this helper to update the version.
Why did you commented it?

Reply:
Because the helper-update-version.sh links with the url = https://api.github.com/repos/duniter/cesium/releases/latest and this version is not here, it makes an error. It is a Pre-release.

Reply:
https://github.com/duniter/cesium/releases/ to see, the latest is 1.6.5 here: https://api.github.com/repos/duniter/cesium/releases/latest

Reply:
What's the issue?

Reply:
Ok, I see. the v1.6.7 brought the read-only mode.
And you can not do the update with the script.
I usually move the release to "latest", then run the helper.
I can do that because I have permissions on the repository.
But, for people who don't, that would be great if the script could look for the latest unstable release. An option allowing to choose between the stable and the unstable would be nice!

Reply:
You're right, but I am not sure to know how to it.
 I am refactoring the code with the new examples_ynh, and have add change_url script also. I am waiting for the jenkins results...

Reply:
Can we focus this PR on #19?
Can you open an other PR to refactor the package with latests standards?
That would ease the review, make #19 ends-up earlier in `master` for everybody, and not mix too many potential bug altogether.
Thanks.

Reply:
Oh, I am sorry, I have push all my changes on the same PR, it is not a good practice. I thought it was a good idea refactoring the code also...
Here is my result with Jenkins: https://ci-apps-dev.yunohost.org/jenkins/job/cesium_ynh%20(rodinux)/4/console
But tell me how I can change, making another fork with only the first steps, and others PR after about change the syntax (for ynh_create_permissions, I also change the sed commands with ynh_replace_string and also the syntax for nginx.conf) ?

Reply:
I pushed to master the update to v1.6.7.

About getting the latest pre-release with the helper: #21.

Reply:
To create an other PR, you should be able to use an other branch on your fork to create a PR on this repository.

Reply:
Ok, I have made a branch test where the helper-update-version.sh is uncommented. I think it is good. I change the version to ~ynh3 also. Can you see all the changes in the code ? Before make a PR I have to try updating from the current stable version...

Reply:
Can you clean the branch which this PR is based on: `master`.
You will need to rebase, then push force.

Reply:
> Can you clean the branch which this PR is based on: `master`.
> You will need to rebase, then push force.
Its a PR based on  `test` branch, do you prefer I merge it on `master branch` ?
I have still a result with jenkins here: https://ci-apps-dev.yunohost.org/jenkins/job/cesium_ynh%20(rodinux)/5/console
Perhaps it could be better...

Reply:
I had made a mistake changing the manifest.json `version ~ynh1` but I put back as the original.

Reply:
Let's continue this discussion on #22.